tests/bulkwritecommand/bulkwritecommand-updateOne_error-006.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+MongoDB\Driver\BulkWriteCommand::updateOne() prohibits PackedArray for document values
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$bulk = new MongoDB\Driver\BulkWriteCommand;
+
+echo throws(function() use ($bulk) {
+    $bulk->updateOne(NS, MongoDB\BSON\PackedArray::fromPHP([]), []);
+}, MongoDB\Driver\Exception\UnexpectedValueException::class), "\n";
+
+echo throws(function() use ($bulk) {
+    $bulk->updateOne(NS, [], [], ['collation' => MongoDB\BSON\PackedArray::fromPHP([])]);
+}, MongoDB\Driver\Exception\UnexpectedValueException::class), "\n";
+
+// @TODO: ALMOST: Got MongoDB\Driver\Exception\InvalidArgumentException - expected MongoDB\Driver\Exception\UnexpectedValueException
+// Expected "hint" option to yield string or document but got "array"
+echo throws(function() use ($bulk) {
+    $bulk->updateOne(NS, [], [], ['hint' => MongoDB\BSON\PackedArray::fromPHP([])]);
+}, MongoDB\Driver\Exception\UnexpectedValueException::class), "\n";
```

Comment:
The error message is different:
- Collection BulkWrite: `MongoDB\BSON\PackedArray cannot be serialized as a root document`
- Client BulkWriteCommand: `Expected "hint" option to yield string or document but got "array"`